code/ai/aicode.cpp

```diff
@@ -1257,41 +1304,29 @@ void ai_turn_towards_vector(vec3d *dest, object *objp, vec3d *slide_vec, vec3d *
 			delta_bank = -delta_bank;
 	}
 
-	//	Dave Andsager: The non-indented lines here are debug code to help you track down the problem in the physics
-	//	that is causing ships to inexplicably rotate very far.  If you see the message below in the log, set the next statement to be
-	//	the one marked "HERE".  (Do this clicking the cursor there, then right clicking.  Choose the right option.)
-	//	This will allow you to rerun vm_angular_move_forward_vec() with the values that caused the error.
-#ifndef NDEBUG
```

Comment:
Yes, double checking everything, sorry.  This was not an issue during your work, right?

Reply:
Ah! I had meant to comment on this change. I have never seen the mprintf later in the function that this refers to, and I'm almost certain this was used for debugging something during development. I don't mind leaving it in just in case, but after working so much on this function it really becomes a readability eyesore for me.

Reply:
I honestly don't think it's a big deal.  I don't like these old debug statements or things that tell you to "ask Allender" and things.  But someone above my pay grade may say something.  No one has taken a deeper look at this than you, so I imagine you would have found the problem if it still existed.